### PR TITLE
Check alerts in Elasticsearch

### LINKED ISSUE #472: Compare generated alerts with alerts indexed in Elasticsearch
### Objective
Create a mechanism to verify that the alerts generate when creating or modifying files are correctly indexed in Elasticsearch.

### Tasks
- [x] Query to elasticsearch
- [x] Tasks to lauch query to Elasticsearch sever

Regards,
Daniel Folch

### PR DESCRIPTION
**Pull request by DFolchA**
Hello team,

This PR closes #472 , In this PR I have created a script that check that the expected alerts are included in Elasticsearch and the necessary tasks to launch this script.

Regards, 
Daniel Folch

### CODE REVIEW

File: tests/system/fim/roles/aws/tasks/generate_environment_template.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (jm404) [inline tests/system/fim/roles/aws/tasks/generate_environment_template.yml]: This file is not required anymore, please delete it.

File: tests/system/fim/roles/aws/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+---
```
Comment (jm404) [inline tests/system/fim/roles/aws/tasks/main.yml]: This file is not required anymore, please delete it

File: tests/system/fim/roles/aws/templates/environment_template.j2 [hunk 3]
```diff
@@ -0,0 +1,1503 @@
+AWSTemplateFormatVersion: 2010-09-09
```
Comment (jm404) [inline tests/system/fim/roles/aws/templates/environment_template.j2]: This file is not required anymore, please delete it

File: tests/system/fim/roles/aws/tasks/main.yml (hunk 2 above)
Comment (DFolchA) [reply tests/system/fim/roles/aws/tasks/main.yml]: Done in https://github.com/wazuh/wazuh-qa/pull/482/commits/a42a0040017e5cc3755ad594fa757f0e55198723 .

File: tests/system/fim/roles/aws/tasks/generate_environment_template.yml (hunk 1 above)
Comment (DFolchA) [reply tests/system/fim/roles/aws/tasks/generate_environment_template.yml]: Done in https://github.com/wazuh/wazuh-qa/pull/482/commits/a42a0040017e5cc3755ad594fa757f0e55198723 .

File: tests/system/fim/roles/aws/templates/environment_template.j2 (hunk 3 above)
Comment (DFolchA) [reply tests/system/fim/roles/aws/templates/environment_template.j2]: Done in https://github.com/wazuh/wazuh-qa/pull/482/commits/a42a0040017e5cc3755ad594fa757f0e55198723 .

File: tests/system/fim/common_tasks/verify_alerts.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+---  
+  - name: "Verify alert log"
+    script: "verify_alerts_json.py -f {{ file_list_path }} -e {{ syscheck_event }}"
```
Comment (manuasir) [inline tests/system/fim/common_tasks/verify_alerts.yml]: Hey @DFolchA thank you for adding the previously requested changes. I'm wondering where these `file_list_path` and `syscheck_event` variables are coming from. I assume that the `private_ip` comes from the generated Ansible hostfile, but not these two.

File: tests/system/fim/common_tasks/verify_alerts_elasticsearch.py [hunk 5]
```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/python3
+
+import argparse
+import json
+from elasticsearch import Elasticsearch
+
+def setElasticsearch( ElasticIP):
```
Comment (manuasir) [inline tests/system/fim/common_tasks/verify_alerts_elasticsearch.py]: Please add the proper comments to these methods and ensure this script is `Flake8` compliant.

File: tests/system/fim/common_tasks/verify_alerts.yml (hunk 4 above)
Comment (DFolchA) [reply tests/system/fim/common_tasks/verify_alerts.yml]: Thes two variables should be provided to the tasks by the playbook calling them.
`file_list_path` is the path to the file containing all the files created/modified it should be created when creating the files and provided to the machines that need it.
`syscheck_event` is the kind of alert event we are looking for: added, modified or deleted; this is one of the parameters of the test.

File: tests/system/fim/common_tasks/verify_alerts_elasticsearch.py (hunk 5 above)
Comment (DFolchA) [reply tests/system/fim/common_tasks/verify_alerts_elasticsearch.py]: Fixed in https://github.com/wazuh/wazuh-qa/pull/482/commits/3098c93c91f79f9c2a854229fe9c38ed0162ac47

```
flake8.checker            MainProcess     66 INFO     Making checkers
flake8.checker            MainProcess     68 INFO     Checking 1 files
flake8.main.application   MainProcess     80 INFO     Finished running
flake8.main.application   MainProcess     80 INFO     Reporting errors
flake8.main.application   MainProcess     80 INFO     Found a total of 0 violations and reported 0
```

**Pull request merged**